pkg/client/client.go

```diff
@@ -251,15 +307,13 @@ func NewDynatraceClient(environmentURL string, token string, opts ...func(dynatr
 		log.Warn("You are using an insecure connection (%s). Consider switching to HTTPS.", parsedUrl.Scheme)
 	}
 
-	if !isNewDynatraceTokenFormat(token) {
-		log.Warn("You used an old token format. Please consider switching to the new 1.205+ token format.")
-		log.Warn("More information: https://www.dynatrace.com/support/help/dynatrace-api/basics/dynatrace-api-authentication")
+	if httpClient == nil {
+		httpClient = &http.Client{}
```

Comment:
does that case still happen and should we even support it? Or rather error out if no client was given?

Reply:
I don't have a strong opinion on that.
I think it is easy to handle so let's keep it?

Reply:
fine by me, was just wondering